pinot-common/src/main/java/com/linkedin/pinot/common/utils/TarGzCompressionUtils.java

```diff
@@ -165,11 +167,11 @@ private static void addFileToTarGz(TarArchiveOutputStream tOut, String path, Str
    * @throws IOException
    * @throws FileNotFoundException
    *
-   * @return  The {@link List} of {@link File}s with the untared content.
+   * @return The {@link List} of {@link File}s with the untared content.
    * @throws ArchiveException
    */
-  public static List<File> unTar(final File inputFile, final File outputDir) throws FileNotFoundException, IOException,
-      ArchiveException {
+  public static List<File> unTar(final File inputFile, final File outputDir)
+      throws FileNotFoundException, IOException, ArchiveException {
 
     String outputDirectoryPath = outputDir.getCanonicalPath();
     LOGGER.debug(String.format("Untaring %s to dir %s.", inputFile.getAbsolutePath(), outputDirectoryPath));
```

Comment:
we have excessive usage of String.format, can we use slf4j api and remove the usage of String.format. Should avoid unnecessary string creation

Reply:
Sure, I can clean this up.